src/Optimization/hiopDualsUpdater.hpp

```diff
@@ -148,18 +178,36 @@ class hiopDualsLsqUpdate : public hiopDualsUpdater
         nlp_->log->printf(hovScalars,
                           "will not use lsq dual initial point since its norm (%g) is larger than "
                           "the tolerance duals_lsq_ini_max=%g.\n",
-                          ynrm, lsq_dual_init_max);
+                          ynrm,
+                          lsq_dual_init_max);
       }
     }
     return bret;
   }
 protected:
-  //method called by both 'go' and 'computeInitialDualsEq'
+  /// Helper method doing the work for both `go` and `computeInitialDualsEq`
```

Comment:
maybe it's a good time for us to rename `computeInitialDualsEq` as `compute_initial_duals`